### ISSUE TITLE
Ошибка при работе с cubegame.

### ISSUE THREAD

**Issue by just-a-tractor** [cepreu2github/cubegame#1]
Здравствуйте, меня зовут Алексей. Я учусь в it школе samsung, сейчас изучаю libGDX. Написать вам посоветовал преподаватель. Пытался понять основы работы библиотеки на примере, но каждый раз возникает ошибка- 
"Caused by: java.io.FileNotFoundException: SPACE_FR.PNG". 
Текстура лежит там же, где и модель- в assets. Не знаю что делать, все перепробовал. Но без использования модели (с зеленым тетраэдром) всё работает. Подскажите, в чем может быть проблема. Использую android Studio.

**Comment by cepreu2github** [comment]
https://habr.com/ru/post/276139/

Проблема описана, решение тоже. Поищите в статье строку "SPACE_FR.PNG".

**Comment by cepreu2github** [comment]
Извините, давно отошел от Android-разработки. Даже Android Studio сейчас нет установленного, чтобы быстренько проверить код. Единственное предположение: а вы точно назвали в правильном регистре? Android чувствителен к регистру, должно быть именно так капсом и имя файла и расширение.

**Comment by just-a-tractor** [comment]
Да, я файл скачал и расширение большими буквами переписал. Даже пробовал переименовывать, копируя название из текста ошибки, из текстового представления модели, из примера- ничего.
